test/images/Makefile

```
@@ -15,6 +15,7 @@
 include  ../../hack/make-rules/Makefile.manifest
 
 REGISTRY ?= gcr.io/kubernetes-e2e-test-images
+REMOTE_DOCKER_URL ?= ""
```

This was being set to `""` instead of the empty string
```suggestion
REMOTE_DOCKER_URL ?=
```